src/components/body/CryptoList.tsx

```diff
@@ -1,47 +1,58 @@
 import React from 'react';
-import {Text} from 'react-native';
-import AntDesign from 'react-native-vector-icons/AntDesign';
-import {Props} from '../interfaces/Interfaces';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import {colors} from '../../utils/colors';
+import ButonAddCrypto from '../buttons/ButonAddCrypto';
 import NoCryptos from './NoCryptos';
 import {
+  FlatListCustom,
   HorizontalView,
   ImageCrypto,
   ListCryptos,
   TextChange,
+  TextCode,
+  TextName,
   TextPrice,
   ViewEachCryptos,
   ViewHorizontal,
   ViewImageandText,
   ViewPrices,
-  FlatListCustom,
-} from '../styles/CryptoListStyles';
+} from './styles';
 
-const CryptoList = ({cryptos}: Props): JSX.Element => (
+const CryptoList = ({cryptos, navigation}): JSX.Element => (
```

Comment:
missing props type

use FC, from react. and declare the interface of the props in this file:

```
import React, {FC} from 'react';

interface Props = {
  cryptos: TYPE,
  navigation: any
}

const CryptoList = ({cryptos, navigation}): FC<Props>=> (
```